tailscale/rootfs/etc/services.d/cert/run

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/with-contenv bashio
+# ==============================================================================
+# Home Assistant Community Add-on: Tailscale
+# Runs tailscale cert periodically
+# ==============================================================================
+
+# Request TLS certificate if configured
+tailscale_cert() {
+  if bashio::config.true 'request_cert'; then
+    bashio::config.require 'domain_alias'
```

Comment:
we don't have to check this constantly (every 24 hours). If this is kept, we can check it once (in a `cont-init` script).

Reply:
Can we at least auto-check every week? Or, grab the expire date and schedule it in someway. I know we frequently have updates and restart all services (and TS would try to renew the certificates), but some users or at some period we could have it untouched for a long time.

Anyway, if the certificate expires the option for the user is to restart the container, right?

Reply:
Hmm. The issue is renewal. If i have a long uptime without updates or restarts i still want new certs to be fetched automatically. The bash sleep is super light weight so i went with it.